### ISSUE TITLE
Expand to a broader plugin scope for all vendors

### ISSUE THREAD

**Issue by ryanmerolle** [goebelmeier/netbox-cisco-support#9]
There is a huge need in the community to track warranty/support status for all devices (i.e. all manufacturers).

It would be nice to leverage this same data model across all devices(physical & virtual), but still allow users the selectable option (an configurations like cron) to auto-populate devices depending on the vendor support for that via an API or web-scraping.

What does this feature request mean?

1. Rename feature to something like `netbox-support`
2. Rename widgets/cards to remove `Cisco`
3. Allow plugin to work with or without setting up cron or cisco syncs
4. Allow manual setting of EOL/EOS, and other options in a Support page for editing the support plugin model

**Comment by ryanmerolle** [comment]
@goebelmeier I'd love to work with you on this.  I'd even submit a PR given some of the suggestions above.

**Comment by deku-m** [comment]
> @goebelmeier I'd love to work with you on this. I'd even submit a PR given some of the suggestions above.

Hi @ryanmerolle , 

Did you eventually work on an update of this?

**Comment by ryanmerolle** [comment]
No.  Before I work on a PR I like to get acknowledgment from a maintainer.  I do not have interest in forking this plugin at this time and maintaining it alone.
